docs: Guided exercise interactions via CLI (#945)
cosmos/sdk-tutorials: academy/4-my-own-chain/stored-game.md

    )
    ```

## Interact via the CLI

Starport created a set of files for you. It is time to see whether you can already interact with your new checkers blockchain.

1. Start the chain in its shell:

    ```sh
    $ starport chain serve --reset-once
    ...
    🌍 Tendermint node: http://0.0.0.0:26657
    🌍 Blockchain API: http://0.0.0.0:1317
    🌍 Token faucet: http://0.0.0.0:4500
    ```

2. Check the values saved in `NextGame`. Look at the relevant `client/cli` file Starport created to find out what command is relevant. Here it is [`query_next_game.go`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e22/x/checkers/client/cli/query_next_game.go#L14). You can instead also ask the CLI:

    ```sh
    $ checkersd query checkers --help
    ```

    Ok, that is [`show-next-game`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e2251f253288163021c75999709d8c25b402/x/checkers/client/cli/query_next_game.go#L14):

    ```sh
    $ checkersd query checkers show-next-game
    NextGame:
      creator: ""
      idValue: "0"
    ```

>     As expected.

Reviewer: ```suggestion
    This is as expected, because no games have been created yet, so the game counter is still at 0.
```

> 3. If you find the output suitable for reading on the spot but plan to use some scripts, get some information about your options:

Reviewer: ```suggestion
3. The `--output` flag allows you to get your results in a JSON format, which might be useful in case you would like to use a script to parse the information for example. When you use the `--help` flag, you see which flags are available for a specific command:
```

    ```sh
    $ checkersd query checkers --help
    ```

    Ok, that is [`show-next-game`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e2251f253288163021c75999709d8c25b402/x/checkers/client/cli/query_next_game.go#L14):

    ```sh
    $ checkersd query checkers show-next-game
    ```

    Which returns:

    ```
    NextGame:
      creator: ""
      idValue: "0"
    ```

    This is as expected, because no games have been created yet, so the game counter is still at 0.

3. The `--output` flag allows you to get your results in a JSON format, which might be useful in case you would like to use a script to parse the information for example. When you use the `--help` flag, you see which flags are available for a specific command:

    ```sh
    $ checkersd query checkers show-next-game --help
    ```

    Among the output, you see:

    ```
    ...
    -o, --output string   Output format (text|json) (default "text")
    ```

>     Now try again a bit differently:

Reviewer 2: Now, try

>     )

Reviewer 3: I feel like the step 3 should be done as step 1. Now we are using the errors before defining them, with in the end does not matter, but it feels _weird_ to me.

## Interact via the CLI

> Starport creates a set of files for you. It is time to see whether you can already interact with your new checkers blockchain.

Author: "Created" because that was what happened in the previous paragraphs.

    🌍 Token faucet: http://0.0.0.0:4500
    ```

2. Check the values saved in `NextGame`. Look at the relevant `client/cli` file, which Ignite CLI created to find out what command is relevant. Here it is [`query_next_game.go`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e22/x/checkers/client/cli/query_next_game.go#L14). You can also ask the CLI:

    ```sh
    $ checkersd query checkers --help
    ```

    And that is [`show-next-game`](https://github.com/cosmos/b9-checkers-academy-draft/blob/3c69e2251f253288163021c75999709d8c25b402/x/checkers/client/cli/query_next_game.go#L14):

    ```sh
    $ checkersd query checkers show-next-game
    ```

    This returns:

    ```
    NextGame:
      creator: ""
      idValue: "0"
    ```

    This is as expected. No games have been created yet, so the game counter is still at `0`.

> 3. The `--output` flag allows you to get your results in a JSON format, which might be useful if would like to use a script to parse the information. When you use the `--help` flag, you see which flags are available for a specific command:

Author: ```suggestion
3. The `--output` flag allows you to get your results in a JSON format, which might be useful if you would like to use a script to parse the information. When you use the `--help` flag, you see which flags are available for a specific command:
```
@AndrewB9lab
